ambuda/views/proofing/page.py

```diff
@@ -1,5 +1,6 @@
 import difflib
 from pathlib import Path
+import regex
```

Comment:
huh, didn't know this module existed! I always just used `re`.

https://github.com/mrabarnett/mrab-regex

Reply:
`re` doesn't support `\X`.